.github/component_owners.yml

```diff
@@ -114,6 +114,8 @@ components:
     - rauno56
   plugins/node/opentelemetry-instrumentation-router:
     - rauno56
+  plugins/node/opentelemetry-instrumentation-undici:
+    - david-luna
```

Comment:
Not blocking: would be great to have another person who's also interested in being an owner for this :slightly_smiling_face: Commenting here to draw attention to it for anyone interested.

Reply:
the more 👀 the better. thanks @JamieDanielson

Reply:
> I cannot commit to being an owner here right now, but I realize how important this instrumentation is and for now will help keep an eye out for issues... in case that helps 😄

Definitely. Keeping an eye out for issues is always helpful, thanks :slightly_smiling_face:

Reply:
I can be an owner as well.

Reply:
@trentm I'm adding you as an owner then. :)

Reply:
done!